jupyterlab/main.tf

```diff
@@ -49,17 +58,26 @@ resource "coder_script" "jupyterlab" {
   script = templatefile("${path.module}/run.sh", {
     LOG_PATH : var.log_path,
     PORT : var.port
+    BASE_URL : var.subdomain ? "http://localhost:${var.port}/@${data.coder_workspace_owner.me.name}/${data.coder_workspace.me.name}/apps/jupyterlab" : ""
   })
   run_on_start = true
 }
 
 resource "coder_app" "jupyterlab" {
   agent_id     = var.agent_id
-  slug         = "jupyterlab"
+  slug         = "jupyterlab" # sync with with end of subdomain URL
   display_name = "JupyterLab"
-  url          = "http://localhost:${var.port}"
+  url          = var.subdomain ? "http://localhost:${var.port}/@${data.coder_workspace_owner.me.name}/${data.coder_workspace.me.name}/apps/jupyterlab" : "http://localhost:${var.port}"
   icon         = "/icon/jupyter.svg"
-  subdomain    = true
+  subdomain    = var.subdomain
   share        = var.share
   order        = var.order
-}
+
+  dynamic "healthcheck" {
+    for_each = var.subdomain ? toset([true]) : toset([])
+    content {
+      url       = "http://localhost:${var.port}/@${data.coder_workspace_owner.me.name}/${data.coder_workspace.me.name}/apps/jupyterlab"
+      interval  = 6
+      threshold = 10
+    }
+  }
```

Comment:
It looks like it will only work if `subdomain=true` while the `url` is for the case when `subdomain=false`.

Can you please check and also set the healthcheck for both cases?

Reply:
Unfortunately I cannot since I do not have control on our DNS and I do not have time to set it up a new environment privately..

In case, can we keep as it was before this PR, without the healthcheck?

Reply:
Yes, let us drop the healthcheck for now.